eng/common/scripts/modules/Package-Properties.psm1

```diff
@@ -43,6 +48,17 @@ class PackageProps
             $this.pkgChangeLogPath = $null
         }
     }
+    PackageProps(
+        [string]$pkgName,
+        [string]$pkgVersion,
+        [string]$pkgDirectoryPath,
+        [string]$pkgServiceName,
+        [string]$pkgGroupId
```

Comment:
Instead of adding a new constructor you can add a group parameter to the existing constructor and simply default it to empty.

Reply:
Oh, I tried that actually but I was getting errors about the overload of the constructor not found.

Reply:
PowerShell does not seem to have syntax for default parameters in constructor or constructor chaining, so I had to use hidden methods here.